src/main/java/org/jenkinsci/plugins/oic/properties/AllowedTokenExpirationClockSkew.java

```diff
@@ -0,0 +1,59 @@
+package org.jenkinsci.plugins.oic.properties;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import org.jenkinsci.plugins.oic.OicProperty;
+import org.jenkinsci.plugins.oic.OicPropertyDescriptor;
+import org.jenkinsci.plugins.oic.OicPropertyExecution;
+import org.jenkinsci.plugins.oic.OicServerConfiguration;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.pac4j.oidc.config.OidcConfiguration;
+
+/**
+ * Configures the allowed clock skew for token expiration.
+ * This is useful to accommodate for clock differences between the server and the OIDC provider.
+ */
+public class AllowedTokenExpirationClockSkew extends OicProperty {
+    private int value;
```

Comment:
NIT: call it `allowedSkewSeconds` so that it is obvious the user is working in seconds in the back end code (and adjust the other code accordingly)?

Reply:
I propose `valueSeconds` then, I don't like having duplicate wording in casc configuration.